.eslintrc.cjs

```diff
@@ -54,9 +54,8 @@ module.exports = {
                 "newlines-between": "always",
                 "pathGroups": [
                     { "pattern": "#*", "group": "external", "position": "after" },
-                    { "pattern": "#*/*", "group": "external", "position": "after" }
+                    { "pattern": "#*/*", "group": "external", "position": "after" },
```

Comment:
Why do we have imports check by eslint? TS checks imports and this feels redundant

Reply:
This is needed to automatically order and group import statements.